core/loadpoint_session.go

```diff
@@ -57,14 +59,14 @@ func (lp *Loadpoint) stopSession() {
 	if s.Created.IsZero() {
 		return
 	}
-
 	s.Finished = lp.clock.Now()
+
 	if meterStop := lp.chargeMeterTotal(); meterStop > 0 {
 		s.MeterStop = &meterStop
 	}
 
-	if chargedEnergy := lp.GetChargedEnergy() / 1e3; chargedEnergy > s.ChargedEnergy {
-		lp.energyMetrics.Update(chargedEnergy)
```

Comment:
Unrelated and duplicates https://github.com/evcc-io/evcc/pull/19254/files

Reply:
Was meinst du? Hab ich gleich mit aufgeräumt.